### DISCUSSION TITLE
How to setup auth proxy for preview builds?

### DISCUSSION THREAD

**Discussion by sanjeethboddi** [t3-oss/create-t3-turbo#1147] [Q&A]
I've deployed authproxy through vercel.
Used the url from the vercel to setup the redirect uri in discord and .env in the root folder.

when I'm trying to login, I'm getting error "Invalid redirect uri"

`AUTH_REDIRECT_PROXY_URL='https://helium-three.vercel.app/r'`

**Comment by olegkorol** [comment]
Same here. Did you find a solution for this, @sanjeethboddi ?
